pkg/scheduler/factory_test.go

```diff
@@ -433,9 +433,9 @@ func testClientGetPodRequest(client *fake.Clientset, t *testing.T, podNs string,
 			}
 			name := getAction.GetName()
 			ns := a.GetNamespace()
-			if name != podName || ns != podNs {
-				t.Errorf("Expected name %s namespace %s, got %s %s",
-					podName, podNs, name, ns)
+			if name != podName || ns != podNs || podTenant != a.GetTenant() {
+				t.Errorf("Expected name %s namespace %s tenant %s, got %s %s %s",
+					podName, podNs, name, ns, podTenant)
```

Comment:
Seems "podTenant" is missing in the input variable.

Reply:
podTenant is an input arg in the same commit.